pyproject.toml

```diff
@@ -41,7 +41,10 @@ requests = ">=2.20,<2.30"  # TODO: revert upper bound when urllib3 situation sor
 requests-toolbelt = "*"
 importlib-metadata = { version = "*", python = "<3.8" }
 tqdm = ">=4,<5"
-Pillow = "^9.1.1"
+Pillow = [
+    { version = "^9.1.1", python = "<3.8" },
```

Comment:
We're close to being able to [rip out 3.7 support](https://linear.app/kolena/issue/KOL-2344/remove-kolena-support-for-%5Bsoon-to-be-%5Deol-python-37) — [AWS EOLs 3.7 in December](https://aws.amazon.com/blogs/developer/python-support-policy-updates-for-aws-sdks-and-tools/)!